include/deal.II/base/config.h.in

```diff
@@ -448,7 +448,10 @@
 #  define DEAL_II_MPI_VERSION_GTE(major,minor) false
 #endif
 
+#cmakedefine DEAL_II_MPI_WITH_DEVICE_SUPPORT
+#ifdef DEAL_II_MPI_WITH_DEVICE_SUPPORT
 #cmakedefine DEAL_II_MPI_WITH_CUDA_SUPPORT
```

Comment:
Is this now just for backwards compatibility? I don't see anywhere in the library where it is used any more.

Reply:
Only for backward-compatibility.